Source/UI/NewUI/BillingForm/Default Implementation/Elements/Error/DefaultErrorInputLabelStyle.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+
+struct DefaultErrorInputLabelStyle: ErrorInputLabelStyle {
+    var isHidden: Bool
+    var backgroundColor: UIColor
+    var tintColor: UIColor
+    var text: String
+    var font: UIFont
+    var textColor: UIColor
+    var isWarningSympoleOnLeft: Bool
+    var height: Double
+    
+    init(isHidden: Bool = true,
+         backgroundColor: UIColor = .white,
+         tintColor: UIColor = .tallPoppyRed,
+         text: String = "",
+         font: UIFont = UIFont(graphikStyle: .medium, size: 13),
```

Comment:
Do you want to put all the magic numbers, whitespaces in one constants file to easily manage and refer ?

Reply:
they are only default value can be easily overridden